requirements.txt

```diff
@@ -29,7 +29,7 @@ importlib_resources
 # webapp login
 authlib
 flask_login
-flask-ldapconn
+flask-ldapconn @ git+https://github.com/Clinical-Genomics/flask-ldapconn.git@v0.10.2
```

Comment:
I have a feeling that they merged my PR on the original repo without testing it, so I'm a bit worried that this change might brake the LDAP connection functionality.. I think we should at least trying the LDAP connection following the simple Futurama tutorial?

Reply:
💯 Sure, I was certain you had tested it before PRing an external repo.

Reply:
I've tried to use the original tests of that repo, but I couldn't make it work

Reply:
👍🏻 but let's open an issue so we remember to switch to the original repo PyPi once they update